conans/client/api/conan_api.py

```diff
@@ -268,23 +269,100 @@ def _get_revisions(self, ref, remote=None):
                     "time": from_timestamp_to_iso8601(timestamp)
                 }
                 results.append(result)
-            return results
+
+        return results
 
     @api_method
     def get_package_revisions(self, reference, remote=None):
+        """
+        Get all the recipe revisions given a reference from cache or remote.
+
+        :param reference: `PackageReference` without the revision
+        :param remote: `Remote` object
+        :return: `dict` with all the results, e.g.,
+                 {"remote": "my_remote_name",  # or None
+                  "reference": "libyaml/0.2.5#80b7cbe095ac7f38844b6511e69e453a:ef93ea55bee154729e264db35ca6a16ecab77eb7",
+                  "results": [{"revision": "80b7cbe095ac7f38844b6511e69e453a",
+                               "time": "2021-07-20 00:56:25 UTC"}, ...]}
+        """
```

Comment:
please reformat the example to be more easily read (Also in `get_recipe_revisions` and `get_package_ids`:

```
:return: `dict` with all the results, e.g.,
  {
    "remote": "my_remote_name",  # or None
    "reference": "libyaml/0.2.5#80b7cbe095ac7f38844b6511e69e453a:ef93ea55bee154729e264db35ca6a16ecab77eb7",
    "results": [
      {
        "revision": "80b7cbe095ac7f38844b6511e69e453a",
        "time": "2021-07-20 00:56:25 UTC"
      },
      ...
    ]
  }
```